sdk/program/src/syscalls.rs

```diff
@@ -0,0 +1,123 @@
+use crate::{
+    instruction::{AccountMeta, ProcessedSiblingInstruction},
+    pubkey::Pubkey,
+};
+
+#[cfg(target_feature = "static-syscalls")]
+macro_rules! define_syscall {
+    (fn $name:ident($($arg:ident: $typ:ty),*) -> $ret:ty) => {
+		#[inline]
```

Comment:
Why inline?

Reply:
The compiler is probably smart enough to inline without, but essentially we want the syscall to result in one CALL_IMM (to the hash), not two (to the generated function then to the hash).